src/main/java/org/opentripplanner/routing/bike_rental/BikeRentalStation.java

```diff
@@ -99,4 +102,17 @@ public BikeRentalStation clone() {
     public String getName() {
         return name.toString(locale);
     }
+
+    public static class RentalUris {
+
+        public final String android;
+        public final String ios;
+        public final String web;
```

Comment:
JavaDoc on public fields. I assume these are nullable? Maybe a @Nullable is enough if the class JavaDoc can document these "together".

Reply:
I've added documentation to the class and fields.